__tests__/mockjs_test.res

```diff
@@ -104,7 +106,9 @@ let _ = {
       let fn = MockJs.fn(mockFn)
 
       let before = call(fn, 10)
-      mockFn |> MockJs.mockImplementation((. a) => Js.Undefined.return(string_of_int(a))) |> ignore
+      mockFn
+      -> MockJs.mockImplementation((. a) => Js.Undefined.return(string_of_int(a)))
+      -> ignore
```

Comment:
I don't think having an API that mixes pipe-first and pipe-last is a good idea since it will be hard to remember which APIs use which kind of pipe. I don't particularly care which is used, as long as it's used consistently.

Reply:
I updated the API and test to data-first semantics